### ISSUE TITLE
OpenAPI spec does not document tasks generated by a GET to /declare

### ISSUE THREAD

**Issue by mstokes-f5** [F5Networks/f5-appsvcs-extension#321]
### Environment
 * Application Services Version: 3.21.0-4
 * BIG-IP Version: N/A

### Summary
The [OpenAPI docs for the /task endpoint on AS3](https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/refguide/apidocs.html#tag/Task) do not document tasks that result from doing a `GET` on the `/declare` endpoint. These tasks have the declaration as an item in the  `results` array.

### Steps To Reproduce
1. Have AS3 managed applications on the box (need something to get back from `/declare`)
2. Do a `GET` on the `/declare` endpoint
3. Do a `GET` on the `/task` endpoint

### Expected Behavior
The results of the `GET` on the `/task` endpoint should be documented.

### Actual Behavior
A result like the below is returned which does not match the OpenAPI spec:
```json
{
  "items": [
    {
      "id": "4938b176-8836-47cf-bc5f-06dccd03d423",
      "results": [
        {
          "class": "ADC",
          "schemaVersion": "3.0.0",
          "foo": {
            "class": "Tenant",
            "bar": {
              "class": "Application",
              "template": "udp",
              "serviceMain": {
                "class": "Service_UDP",
                "virtualAddresses": [
                  "192.0.2.1"
                ],
                "virtualPort": 5555,
                "pool": "bar_Pool1"
              },
              "bar_Pool1": {
                "class": "Pool",
                "monitors": [
                  "icmp"
                ],
                "members": [
                  {
                    "serverAddresses": [
                      "192.0.2.2"
                    ],
                    "servicePort": 5555
                  }
                ]
              },
              "constants": {
                "class": "Constants",
                "fast": {
                  "template": "examples/simple_udp_defaults",
                  "setHash": "c2952188146772dc1adbcde6d7618b330cccd5d18c0c20952b2bd339b8889c87",
                  "view": {
                    "tenant_name": "foo",
                    "application_name": "bar",
                    "virtual_address": "192.0.2.1",
                    "virtual_port": 5555,
                    "server_addresses": [
                      "192.0.2.2"
                    ],
                    "service_port": 5555
                  },
                  "lastModified": "2020-08-07T20:51:46.749Z"
                }
              }
            }
          },
          "id": "fast%create%foo%bar%3f7d3920-96be-4ec9-89c2-a26e07ca5205",
          "updateMode": "selective",
          "controls": {
            "archiveTimestamp": "2020-08-07T20:51:50.394Z"
          }
        }
      ],
      "declaration": {}
    }
  ]
}
```

### Possible Solutions
This behavior should be documented. It would also be nice to be able to easily identify what type of task is being dealt with when trying to parse the results of the `/task` endpoint, but that is a different issue/request.

**Comment by dstokesf5** [comment]
Added to our internal product backlog as AUTOTOOL-1795.

**Comment by dstokesf5** [comment]
This was resolved with the 3.31.0 release of AS3.
